extensions/amp-story/1.0/amp-story.js

```diff
@@ -165,6 +171,14 @@ const Attributes = {
  */
 const PAGE_LOAD_TIMEOUT_MS = 5000;
 
+/**
+ * The duration of time (in milliseconds) after a story has automatically
+ * advanced during which the user will not be able to manually advance, to
+ * prevent accidental duplicate advances.
+ * @private @const {number}
+ */
+const NAVIGATION_DEBOUNCE_THRESHOLD_MS = 750;
```

Comment:
Personal opinion backed by not much: given that other stories platform like Instagram/TikTok/Snapchat don't do anything here and let users solve this on their own, I think this threshold should be a lot lower.
There is some consensus on the web around a magic ~300ms number, that has been used for web perf recommendations (all actions should be under this) or by browsers (timer the browser used to wait to see if a tap could become a double tap).
Up to you though

Reply:
300ms works for me.  I had it as that initially and lengthened it based on just feeling it out, but I think what you say makes sense.  Of course, we can always adjust this number over time if it's really that big of an issue, but erring on the side of a short threshold sounds like the right principle here.